TMDBApp/TMDBApp/General.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Geral.swift
```

Comment:
I think its a good way to have some code that I'll reuse on the project.
I'm not sure if its a good practice but for now I don't see a better way.